src/executor/FaabricExecutor.cpp

```diff
@@ -150,7 +132,7 @@ std::string FaabricExecutor::processNextMessage()
     faabric::Message msg = currentQueue->dequeue(timeoutMs);
 
     std::string errorMessage;
-    if (msg.isflushrequest()) {
+    if (msg.type() == faabric::Message_MessageType_FLUSH) {
```

Comment:
Rather than having boolean flag properties like `isflushrequest` we should use the existing message type field.